### PR TITLE
#182: Resize parcels table column width

### PR DESCRIPTION
**Pull request by kaydentsoftwire**
## What's changed
Parcel table column widths are adjusted - full name and status columns proportionally larger, accounting for a smaller screen, and ensuring that the icon column shrinks on a smaller screen

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| image image | image image |

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

### CODE REVIEW

File: src/app/parcels/parcelsTable/styles.ts [hunk 1]
```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
 
 export const parcelTableColumnStyleOptions = {
     iconsColumn: {
-        width: "4rem",
+        maxWidth: "4rem",
+        minWidth: "0.7rem",
+        grow: 0.5,
     },
```
Comment (kaydentsoftwire) [inline src/app/parcels/parcelsTable/styles.ts:17]: Allows icons column to shrink when viewed on a smaller screen

File: src/app/parcels/parcelsTable/styles.ts [hunk 2]
```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
...
     fullName: {
-        grow: 1.3,
-        minWidth: "8rem",
+        grow: 1.2,
+        minWidth: "6.5rem",
     },
```
Comment (kaydentsoftwire) [inline src/app/parcels/parcelsTable/styles.ts:21]: The other grow values for middle columns have shrunk so full name is proportionally larger in relation to the middle columns, but smaller in relation to the last status column to account for a smaller screen

File: src/app/parcels/parcelsTable/styles.ts [hunk 3]
```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
...
-        minWidth: "5rem",
+        grow: 0.5,
+        minWidth: "4rem",
     },
     lastStatus: {
         grow: 3,
         maxWidth: "30rem",
+        minWidth: "10rem",
     },
 };
```
Comment (kaydentsoftwire) [inline src/app/parcels/parcelsTable/styles.ts:51]: gave last status a min width to make it larger on a smaller screen, and adjusted the sizing of the other columns to be proportionally smaller

File: src/app/parcels/parcelsTable/styles.ts [hunk 4]
```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
...
+        grow: 0.6,
         minWidth: "6rem",
     },
     packingDate: {
         hide: 800,
-        grow: 0.8,
-        minWidth: "6rem",
+        grow: 0.7,
+        minWidth: "7rem",
     },
```
Comment (kaydentsoftwire) [inline src/app/parcels/parcelsTable/styles.ts:40]: min width here as increased as the last digit of the year in the date was wrapping

File: src/app/parcels/parcelsTable/styles.ts [hunk 5]
```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
...
         minWidth: "6rem",
     },
     addressPostcode: {
         hide: 800,
-        grow: 0.8,
-        minWidth: "6rem",
+        grow: 0.6,
+        minWidth: "5rem",
+        maxWidth: "6rem",
     },
```
Comment (kaydentsoftwire) [inline src/app/parcels/parcelsTable/styles.ts:31]: this is to keep the postcode size smaller to fit with the character length but also allow a little bit of stretch for wider screens

File: src/app/parcels/parcelsTable/styles.ts (hunk 1 above)
Comment (8kristy) [reply src/app/parcels/parcelsTable/styles.ts:17]: Is that just space around icons or icons themselves?

**Comment by 8kristy** [comment]
It is quite stretched out on the big screen but I don't think we should shrink the smaller columns even more - think it would look a bit silly if we squeezed everything on the left and then had a massive status column. One thing that bothers me is how Postcode and Method are very close together - is it possible to separate them, or somehow distribute the columns evenly?
image

Maybe also give postcode a little bit more room - the longer ones start wrapping around 1220px width
image

Otherwise looks really good 😊

File: src/app/parcels/parcelsTable/styles.ts (hunk 1 above)
Comment (kaydentsoftwire) [reply src/app/parcels/parcelsTable/styles.ts:17]: Just the space around

**Comment by kaydentsoftwire** [comment]
Made the adjustments to the postcode column, so it now looks like this on a larger screen.
image

And this on a smaller screen (no wrapping on postcode):

image

**Review by 8kristy — APPROVED** [review]

**Pull request merged**